### PR TITLE
Add citation details

### PR DESCRIPTION
**Pull request by fjaviersanchez**
Adding badges and citation details to the repo.

### CODE REVIEW

**Comment by dkirkby** [comment]
I suggest slightly stronger wording, e.g. change

> ... please consider citing ... or ...

to

> ...please cite ... and ...

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,42 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
```
Comment (ismael-mendoza) [inline CITATION.cff:2]: is this repeated by mistake?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 Weak lensing fast simulations and analysis for the [LSST](http://www.lsst.org/lsst/) [Dark Energy Science Collaboration](http://www.lsst-desc.org/).
 
 This code was primarily developed to study the effects of overlapping sources on shear estimation,
 photometric redshift algorithms, and deblending algorithms.
 
 User and reference documentation is hosted at http://weaklensingdeblending.readthedocs.io/ and also available as a [single pdf document](https://readthedocs.org/projects/weaklensingdeblending/downloads/pdf/latest/).
 
-[![Documentation Status](https://readthedocs.org/projects/weaklensingdeblending/badge/?version=latest)](https://readthedocs.org/projects/weaklensingdeblending/?badge=latest)
-[![DOI](https://zenodo.org/badge/8727241.svg)](https://zenodo.org/badge/latestdoi/8727241)
+If you use this software please cite [2021JCAP...07..043S](https://ui.adsabs.harvard.edu/abs/2021JCAP...07..043S/abstract) and [this zenodo link](https://zenodo.org/record/3975230#.YVczD2ZKjAM).
```
Comment (ismael-mendoza) [inline README.md:17]: Should everyone cite the Zenodo and the JCAP ? or only people that use the code directly should cite the Zenodo?

**Review by ismael-mendoza — CHANGES_REQUESTED** [review]
Thanks @fjaviersanchez ! Just a couple of small comments.

File: README.md (hunk 2 above)
Comment (ismael-mendoza) [reply README.md:17]: @dkirkby mentioned that people should cite both in this order so this looks good to me

**Review by ismael-mendoza — APPROVED** [review]
Looks good, thanks @fjaviersanchez !

**Pull request merged**